chore: updating dependabot to support gha, python, docker and dev container

rajpalc7 opened the pull request.

rajpalc7 commented:
Hi @wadeBarnes, this is ready for review.

WadeBarnes reviewed (changes requested):
@rajpalc7, You are missing at least one Docker file.

rajpalc7 committed:
chore: updating dependabot to support gha, python, docker and dev container

Signed-off-by: Rajpal Chauhan <rajpal.chauhan@gov.bc.ca>

rajpalc7 commented:
Hi @WadeBarnes, missing docker file has been added now.

WadeBarnes committed:
Pin jsonpickle to 3.0.3

- v3.0.3 is the last version of jsonpickle that contains the setup.py file required by fpm to build the third party package.
- This is intended as an interim fix for the main branch to allow the workflows to complete.
- Work on the `ubuntu-22.04` branch uses `wheel2deb` to build `deb` packages for the newer python modules.

Signed-off-by: Wade Barnes <wade@neoterictech.ca>

WadeBarnes committed:
Merge pull request #1661 from WadeBarnes/main

Pin jsonpickle to 3.0.3
